sim/paladin/items.go

```diff
@@ -49,36 +53,104 @@ func init() {
 		})
 	})
 
-	
+	// https://www.wowhead.com/classic/item=230272/scrolls-of-blinding-light
+	// Use: Energizes a Paladin with light, increasing melee attack speed by 25%
+	// and spell casting speed by 33% for 20 sec. (2 Min Cooldown)
+	core.NewItemEffect(ScrollsOfBlindingLight, func(agent core.Agent) {
+		character := agent.GetCharacter()
+
+		duration := time.Second * 20
+
+		aura := character.GetOrRegisterAura(core.Aura{
+			Label:    "Blinding Light",
+			ActionID: core.ActionID{SpellID: 467522},
+			Duration: duration,
+			OnGain: func(aura *core.Aura, sim *core.Simulation) {
+				character.MultiplyAttackSpeed(sim, 1.25)
+				character.MultiplyCastSpeed(1.33)
+			},
+			OnExpire: func(aura *core.Aura, sim *core.Simulation) {
+				character.MultiplyAttackSpeed(sim, 1.0/1.25)
+				character.MultiplyCastSpeed(1.0 / 1.33)
+			},
+		})
+
+		spell := character.GetOrRegisterSpell(core.SpellConfig{
+			ActionID: core.ActionID{ItemID: ScrollsOfBlindingLight},
+			Flags:    core.SpellFlagNoOnCastComplete | core.SpellFlagOffensiveEquipment,
```

Comment:
Would you mind adding `ProcMask: ProcMaskEmpty` here?

Reply:
sure